e2e/test-util.ts

```diff
@@ -94,3 +94,43 @@ export async function deleteVFolderAndVerify(page: Page, folderName: string) {
   await page.getByRole('button', { name: 'Delete forever' }).click();
   await expect(page.locator('#trash-bin-folder-storage').getByText('e2e-test-folder', { exact: true })).toBeHidden();
 }
+
+export async function createSession(page:Page,sessionName:string){
+  await navigateTo(page,'job');
+  await page.locator('#launch-session').filter({ hasText: 'Start' }).click()
+
+ // Session launch
+  const sessionNameInput =  page.locator('#sessionName');
+  await sessionNameInput.fill(sessionName);
+  await page.getByRole("button",{name:"Skip to review"}).click();
+
+  await page.locator("button").filter({hasText:"Launch"}).click();
+  await expect(page.locator(".ant-modal-confirm-title")).toHaveText("No storage folder is mounted");
+  await page.getByRole("button",{name:"Start"}).click();
+
+  // Wait for a session to be launched
+  await page.waitForTimeout(10000);   
```

Comment:
It was difficult to me that the sessions created were not immediately reflected on the screen.

So I tried a few things

1. refresh ( use `navigateTo`) -> Not suitable for user scenario testing.
2. “Create session with session name e2e test -> Recreate session with session name e2e test. Check the failure statement that a session has already been created”
I tried to run the test with the above scenario. But It isn't a good fit for user scenario testing either.
3. Increasing the `waitForTimeout` time

I thought option 3 works best and I applied that method

Reply:
Instead of page.waitForTimeout(10000), we can use locator's wait. Please check my recent commit.
```
await expect(session).toBeVisible({timeout: 10000});
```

Reply:
I check it. Create session returns a passed.  But session test fails. Because the timeout (`await page.waitForTimeout(5000);`)was removed and the session termination was not immediately reflected on the screen. 
So I think that Delete session test needs Timeout